packages/page-staking/src/Nomination/NominatedAccount.tsx

```diff
@@ -0,0 +1,350 @@
+// Copyright 2017-2020 @polkadot/app-staking authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import BN from 'bn.js';
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import { Trans } from 'react-i18next';
+import styled from 'styled-components';
+import { DeriveBalancesAll, DeriveStakingAccount, DeriveStakingOverview, DeriveStakerReward } from '@polkadot/api-derive/types';
+import { EraIndex, ValidatorPrefs } from '@polkadot/types/interfaces';
+import { Codec, ITuple } from '@polkadot/types/types';
+import { AddressInfo, AddressMini, AddressSmall, Button, Expander, Menu, Popup, StakingBonded, StakingRedeemable, StakingUnbonding, StatusContext, TxButton } from '@polkadot/react-components';
+import { useAccounts, useApi, useCall, useToggle } from '@polkadot/react-hooks';
+import { FormatBalance } from '@polkadot/react-query';
+import { useTranslation } from '../translate';
+import BondExtra from '../Actions/Account/BondExtra';
+import InjectKeys from '../Actions/Account/InjectKeys';
+import Nominate from '../Actions/Account/Nominate';
+import SetControllerAccount from '../Actions/Account/SetControllerAccount';
+import SetRewardDestination from '../Actions/Account/SetRewardDestination';
+import SetSessionKey from '../Actions/Account/SetSessionKey';
+import Unbond from '../Actions/Account/Unbond';
+import Validate from '../Actions/Account/Validate';
+import { getStakeState, createPayout, StakeState } from '../Actions/Account';
+import useInactives from '../Actions/Account/useInactives';
```

Comment:
I'll need to look at this on-screen, but this looks like a maintenance overhead waiting to happen since we have now basically just duplicated everything we are doing elsewhere in here. This will be missed. I would not advocate doing this at all, this is just going to cause breakages on "o, I didn't look there" down the line.

Reply:
Yes, scrolled down. We certainly do NOT want to be doing what is in here.